batch/operations.tsp

```diff
@@ -0,0 +1,83 @@
+import "@typespec/http";
+import "@typespec/openapi";
+
+import "../common";
+import "./models.tsp";
+
+using TypeSpec.Http;
+using TypeSpec.OpenAPI;
+
+namespace OpenAI;
+
+@route("/batches")
+interface Batches {
+  @post
+  @operationId("createBatch")
+  @tag("Batch")
+  @summary("Creates and executes a batch from an uploaded file of requests")
+  createBatch(
+    /**
+     * The ID of an uploaded file that contains requests for the new batch.
+     *
+     * See [upload file](/docs/api-reference/files/create) for how to upload a file.
+     *
+     * Your input file must be formatted as a [JSONL file](/docs/api-reference/batch/requestInput), and must be uploaded with the purpose `batch`.
+     */
+    input_file_id: string,
+
+    /**
+     * The endpoint to be used for all requests in the batch. Currently `/v1/chat/completions` and `/v1/embeddings` are supported.
+     */
+    endpoint: string,
```

Comment:
```suggestion
    endpoint: "/v1/chat/completions" | "/v1/embeddings"
```
![image](https://github.com/joseharriaga/openai-in-typespec/assets/21109913/3d94f98d-c28f-4523-8490-03db823f7f6f)

Unless this was set as string on purpose

Reply:
Not at all; great catch! I'd originally made it `url` reflexively given the name of `endpoint`, but that of course doesn't work -- I should've caught that it was just a string union like this. I applied your suggestion directly and I'll regenerate the code to match now.